README.md

```diff
@@ -303,6 +303,69 @@ retriable_with_context(:api) do
 end
 ```
 
+## Short Circuiting Retriable While Testing Your App
+
+When you are running tests for your app, it often takes a long time to retry blocks that fail. This is because Retriable will default to 3 tries with exponential backoff. Ideally, your tests will run as quickly as possible.
```

Comment:
just to nitpick some grammar: you should get rid of both commas here.

otherwise thanks; this will definitely help someone else